sites.json

```diff
@@ -1065,6 +1065,17 @@
     },
 
     {
+        "name": "Credit Karma",
+        "url": "https://help.creditkarma.com/hc/en-us/articles/202466954-How-do-I-cancel-my-membership-",
```

Comment:
This should be a direct link to deactivate the account: https://www.creditkarma.com/myprofile/security/deactivate